README.md

```diff
@@ -1,82 +1,28 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
+# Takeaway Challenge
+This is my attempt at the Makers Academy, Week 2, [Takeaway Challenge](https://github.com/makersacademy/takeaway-challenge)
 
- ```
+## Note to reviewer
+The tests for sending a text I have made pending (using ```xit```) as they need a valid mobile number to pass, but do not want texts to be sent each time the tests are run. I have tested them once each and they do send a text when a valid mobile number is given.
 
-Instructions
--------
+If you take the tests off pending then you will see that the tests fail because an invalid number is given.
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+You can test that sending a text works in a REPL by calling ```Order.new.confirm_order(INSERT_VALID_MOB)```.
 
-Task
------
+I think the a couple of the classes could have done with some extraction but, alas, I ran out of time.
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+The ```Order``` class is breaking single responsibility principal since it deals with selecting dishes to add to the order and also calculating the total of the order and checking this against the user's given total. This would be interesting discussion point. Selecting a dish could have been extracted into a separate class if I had time.
 
-```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
+Also, the ```Text``` class calculates the current time and delivery time as well as ordering the API to send the text. A separate ```DeilveryTime``` class could have been created. 
```

Comment:
I think simply calling the class "Delivery" and including Text and Delivery as functions of this class would have been sufficient :)